LINTING.md

```diff
@@ -35,13 +35,16 @@ The following sections outline the minimum, base requirement for linters and for
 
 These tools are a minimum standard and teams may choose to add additional tools where desired.
 
-### Megalinter
+### MegaLinter
 
-:test_tube: **EXPERIMENTAL:** We are currently trialling adoption of Megalinter as a standard. Discuss with your tech lead before adopting with your repos.
+:test_tube: **EXPERIMENTAL:** We are currently trialling adoption of MegaLinter as a standard, see [dis-design-system-go](https://github.com/onsdigital/dis-design-system-go#linting) for an example implementation. Discuss with your tech lead before adopting with your repos.
 
-Megalinter is a docker based linting toolbox that combines many different language specific linters and provides a common summary output.
+[MegaLinter](https://megalinter.io/) is a docker based linting toolbox that combines many different language specific linters and provides a common summary output.
 
-The normal language specific linters should still be used.
+> [!NOTE]
+>
+> * Autofixing should not be enabled in CI
```

Comment:
It would be good to add some rationale here on why, and maybe clarify if we mean all CI (GitHub Actions CI, Concourse CI) etc.